### DISCUSSION TITLE
Should I get another coral usb?

### DISCUSSION THREAD

**Discussion by Alfiegerner** [blakeblackshear/frigate#1708] [Ask A Question] [answered]
Hi, I'm using frigate very successfully with 5 cameras and 1 coral usb. I'm now thinking about about adding some more cameras, and wondering at what point I should consider getting another coral usb? Does anyone have a view on cameras to Coral ratio (understanding it's a bit fuzzy as we're really looking at cameras doing detection). Or what metrics to watch out for to determine that another coral is needed?

Thanks very much.

**Comment by blakeblackshear** [comment]
What is your inference speed?

**Reply by Alfiegerner** [reply]
Inference speed is 11.

**Comment by blakeblackshear** [comment, marked as answer]
With an inference speed of 11, your Coral will top out at 1000/11=90.9 frames per second. If your detection fps is regularly getting close to that, you should first consider tuning motion masks. If those are already properly configured, a second coral may be needed.

**Reply by Alfiegerner** [reply]
Excellent, I know to look for know, thanks very much.
